### PR TITLE
Adds the ol-english element

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_heart: Ready to go.  Adds the ol element to the page-english side.

### CODE REVIEW

File: elements/ol.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>Used to group a list of items in order with a preceding number, letter or Roman numeral.  The preceding number system is indicated in the CSS file using list-style-type to choose between using numbers, letters or Roman numerals, however the actual starting number is in the html file using the start attribute.</p>
```
Comment (werbeckes) [inline elements/ol.html]: Reduce some duplication to shorten and separate into different sentences.  I prefer the oxford comma.

Used to group a list of items in order with a preceding number, letter**,** or Roman numeral.  The preceding number system is indicated in the CSS file using list-style-type**. T**he actual starting number is in the html file using the start attribute.

**Comment by werbeckes** [comment]
Minor wording suggestions.  Otherwise, :+1:

**Comment by kathyoleary** [comment]
did the corrections and ready to go again

**Comment by bigtiger** [comment]
:+1:

File: elements/ol.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>Used to group a list of items in order with a preceding number, letter or Roman numeral.  The preceding number system is indicated in the CSS file using list-style-type.  The actual starting number is in the html file using the start attribute.</p>
```
Comment (ericanaughton) [inline elements/ol.html]: html should be HTML

File: elements/ol.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    </div>
+    <div class="how-to-remember-box how-to-remember-english">
+      <h4>How to Remember</h4>
+      <p>For a list to be in order, it has to be tagged with letters or numbers to keep it in order.</p>
+    </div>
+  </div>
+  <pre class="code-example">
+    <code>
+      <strong>&lt;ol start="10"&gt;</strong>
+          &lt;li&gt;The Tenth Item&lt;/li&gt;
```
Comment (ericanaughton) [inline elements/ol.html]: indentation should only be 2 spaces.

**Comment by savance-ford** [comment]
:+1:

**Comment by savance-ford** [comment]
:+1:  @kathyoleary

**Pull request merged**